tests/devices/qubit_mixed/test_qubit_mixed_measure.py

```diff
@@ -353,6 +302,7 @@ def test_probs_measurement(
     def test_expval_measurement(self, observable, ml_framework, two_qubit_batched_state):
         """Test that expval measurements work on broadcasted state"""
         initial_state = math.asarray(two_qubit_batched_state, like=ml_framework)
+        # observable = math.convert_like(observable, initial_state)
```

Comment:
What's the purpose of this comment?

Reply:
This must be some leftover from debugging forgotten to del.......

Reply:
Will del in next batch of doc commits push